### DISCUSSION TITLE
Libfabric windows support

### DISCUSSION THREAD

**Discussion by theWayofthecode** [ofiwg/libfabric#7023] [Q&A]
Hi,

I want to confirm my assumption that one could use libfabric on windows only via the [fi_netdir(7)](https://ofiwg.github.io/libfabric/v1.13.1/man/fi_netdir.7.html) provider. Is the "windows isn't fully supported" mentioned in the [windows instructions](https://github.com/ofiwg/libfabric#windows-instructions) basically reflecting the limitations of `fi_netdir`?

Best,
Aram

**Comment by j-xiong** [comment]
tcp and udp providers are also available on Windows.

**Reply by theWayofthecode** [reply]
Okay, so it is fi_netdir, tcp and udp then.

**Reply by theWayofthecode** [reply]
More importantly, is RXM working on top of fi_netdir? Basically is there a combination of rxm;fi_netdir that supports the connections-less RMA API?
